crates/iota-transaction-builder/src/builder/mod.rs

```diff
@@ -533,6 +533,34 @@ impl<C, L> TransactionBuilder<C, L> {
     }
 
     /// Split a coin into many.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use iota_graphql_client::Client;
+    /// # use iota_transaction_builder::TransactionBuilder;
+    /// # use iota_types::{Address, ObjectId};
+    ///
+    /// #[tokio::main(flavor = "current_thread")]
+    /// async fn main() -> eyre::Result<()> {
+    ///     let client = Client::new_devnet();
+    ///     let sender = Address::from_hex(
+    ///         "0x611830d3641a68f94a690dcc25d1f4b0dac948325ac18f6dd32564371735f32c",
+    ///     )?;
+    ///     let coin =
+    ///         ObjectId::from_str("0x0b0270ee9d27da0db09651e5f7338dfa32c7ee6441ccefa1f6e305735bcfc7ab")?;
+    ///
+    ///     let mut builder = TransactionBuilder::new(sender).with_client(client);
+    ///     builder.merge_coins(coin_0, [coin_1]);
+    ///         .split_coins(coin, [1000u64, 2000, 3000])
```

Comment:
should be like this?
```suggestion
    ///     builder.split_coins(coin, [1000u64, 2000, 3000])
```

Reply:
yeah